internal/app/referenceserver/server.go

```diff
@@ -283,6 +288,7 @@ func newH1Server(handler http.Handler, listenAddr string, tlsConf *tls.Config) (
 		Handler:           handler,
 		TLSConfig:         tlsConf,
 		ReadHeaderTimeout: 5 * time.Second,
+		ErrorLog:          nopLogger(),
```

Comment:
This is an unrelated improvement. I would occasionally see log lines from the reference server if something failed during TLS handshake (some cases in the connect-kotling conformance tests were seeing timeouts happen during handshake, so the server gets an unexpected EOF and was logging it to stderr).

So this just silences that log output. If we re-enable it in the future, it should use the `errWriter` passed in (instead of the default logger, which writes to `os.Stderr`).